Boolean.CSharp.Test/CoreTests.cs

```diff
@@ -1,24 +1,61 @@
 ﻿using Boolean.CSharp.Main;
+using Newtonsoft.Json;
 using NUnit.Framework;
 
 namespace Boolean.CSharp.Test
 {
     [TestFixture]
     public class CoreTests
     {
-        private Core _core;
+        private Customer _customer;
+        //private BankStatement _bankStatement;
 
         public CoreTests()
         {
-            _core = new Core();
-
+            _customer = new Customer();
+            //_bankStatement = new BankStatement();
         }
 
         [Test]
-        public void TestQuestion1()
+        public void addAccountTest()
         {
-
+            SavingsAccount newSavings = (SavingsAccount)_customer.createAccount(762308, accountType.savings);
+            CurrentAccount newCurrent = (CurrentAccount)_customer.createAccount(556223, accountType.current);
+            Assert.That(_customer.Accounts.Contains(newSavings));
+            Assert.That(_customer.Accounts.Contains(newCurrent));
+        }
+        [Test]
+        public void depositWithdrawTest()
+        {
+            SavingsAccount savingsAccount = (SavingsAccount)_customer.createAccount(123456, accountType.savings);
+            CurrentAccount currentAccount = (CurrentAccount)_customer.createAccount(642758, accountType.current);
+            savingsAccount.deposit(0f);
+            currentAccount.deposit(0f);
+            Assert.That(savingsAccount.Balance, Is.EqualTo(0f));
+            Assert.That(currentAccount.Balance, Is.EqualTo(0f));
+            savingsAccount.deposit(1000f);
+            currentAccount.deposit(50f);
+            Assert.That(savingsAccount.Balance, Is.EqualTo(1000f));
+            Assert.That(currentAccount.Balance, Is.EqualTo(50f));
```

Comment:
you are not testing whether the accounts are storing the correct valid Transaction objects taht should be created during withdraw/deposit